src/Algebra/Bundles.agda

```diff
@@ -670,6 +670,25 @@ record CommutativeSemiring c ℓ : Set (suc (c ⊔ ℓ)) where
     }
 
 
+record CommutativeCancellativeSemiring c ℓ : Set (suc (c ⊔ ℓ)) where
+  infixl 7 _*_
+  infixl 6 _+_
+  infix  4 _≈_
+  field
+    Carrier                           : Set c
+    _≈_                               : Rel Carrier ℓ
+    _+_                               : Op₂ Carrier
+    _*_                               : Op₂ Carrier
+    0#                                : Carrier
+    1#                                : Carrier
+    isCommutativeCancellativeSemiring : IsCommutativeCancellativeSemiring
+                                          _≈_ _+_ _*_ 0# 1#
```

Comment:
The line is already breaching the character limit, so you might as well put the parameters on the same line.